build.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
   compileSdkVersion = 26
-  buildToolsVersion = '26.0.2'
-  buildGradleVersion = '3.0.1'
+  buildToolsVersion = '28.0.3'
```

Comment:
This doesn't need to be specified anymore. AGP will select and install the version it needs.

Reply:
sweet